Libplanet.SDK.Action/Action/ActionBase.cs

```diff
@@ -11,14 +11,14 @@ namespace Libplanet.SDK.Action
     public abstract partial class ActionBase : IAction
     {
         public IValue PlainValue => Dictionary.Empty
-            .Add("name", _name ?? throw new NullReferenceException())
-            .Add("call", _call ?? "Execute")
-            .Add("args", _args ?? Null.Value);
+            .Add("type_id", _name ?? throw new NullReferenceException())
+            .Add("call", _call ?? throw new NullReferenceException())
+            .Add("args", _args ?? throw new NullReferenceException());
 
         public void LoadPlainValue(IValue plainValue)
         {
             Dictionary dict = (Dictionary)plainValue;
-            _name = (Text)dict["name"];
+            _name = (Text)dict["type_id"];
             _call = (Text)dict["call"];
             _args = dict["args"];
         }
```

Comment:
How about a pattern like below?

```csharp
        void IAction.LoadPlainValue(IValue plainValue)
        {
            if (_isLoaded)
            {
                throw new InvalidOperationException("Action is already loaded.");
            }

            Dictionary dict = (Dictionary)plainValue;
            _name = (Text)dict["type_id"];
            _exec = (Text)dict["exec"];
            _args = dict["args"];
            _isLoaded = true;
        }

```

It has two advantages: limited access and avoiding duplicate calls.